akka-docs/src/main/paradox/stream/operators/Source/completionStageSource.md

```diff
@@ -6,6 +6,21 @@ Streams the elements of an asynchronous source once its given *completion* opera
 
 ## Signature
 
+@@@ div { .group-scala }
+
+@@snip[Source.scala](/akka-stream/src/main/scala/akka/stream/javadsl/Source.scala) { #completionStageSource }
+
+@@@
+
+@@@ div { .group-java }
+
+```java
+public static <T,M> Source<T, CompletionStage<M>> completionStageSource(CompletionStage<Source<T, M>> completionStage)
```

Comment:
There's no java implementation of the method so I manually created this (but I suspect we don't want it).

Reply:
What we've been doing for other operators is linking to the scala/javadoc methods instead of showing snippets - I guess we should document that in the umbrella issue